test/pkg/test/service_export_import.go

```diff
@@ -0,0 +1,42 @@
+package test
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
+)
+
+func (env *Framework) GetServiceExportAndServiceImportByService(service *v1.Service) (*v1alpha1.ServiceExport, *v1alpha1.ServiceImport) {
```

Comment:
Is there a reason you use `Getxxxxxx()` ?

Reply:
Maybe change to  `CreateServiceExportAndServiceImportByService` ?